src/utils.js

```diff
@@ -13,3 +13,18 @@ export function useOldConfig(newConfig) {
     USE_CAMERA: equipment.camera ? true : false,
   };
 }
+
+/**
+ * Get a query parameter out of the window"s URL
+ * @param {*} variable The key to parse
+ */
+// TODO 199: Can this just use URLSearchParams?
+// TODO: MOve this out of JsPsych
+export function getQueryVariable(variable) {
+  const query = window.location.search.substring(1);
+  const vars = query.split("&");
+  for (let i = 0; i < vars.length; i++) {

```

Comment:
Look at forEach Array method -- I think it makes sense to use here if you think it will make the code more readable. Obv just a style choice though, feel free to ignore!

Reply:
Yeah this is an old function... I'm thinking [URLSearchParams](https://developer.mozilla.org/en-US/docs/Web/API/URLSearchParams) are in order but I'm not sure if they work with Electron yet